### ISSUE TITLE
ValueError: Timezone offset does not match system offset: -7200 != -10800. Please, check your config files.

### ISSUE THREAD

**Issue by tsouza85** [morpheus65535/bazarr#641]
**Describe the bug**
I installed Bazaar and when starting from cmd this error:

```
Bazarr starting...
Traceback (most recent call last):
  File "c:\bazarr\bazarr\main.py", line 54, in <module>
    from scheduler import *
  File "c:\bazarr\bazarr\scheduler.py", line 66, in <module>
    if str(get_localzone()) == "local":
  File "c:\bazarr\bazarr\../libs/tzlocal\win32.py", line 95, in get_localzone
    utils.assert_tz_offset(_cache_tz)
  File "c:\bazarr\bazarr\../libs/tzlocal\utils.py", line 38, in assert_tz_offset
    raise ValueError(msg)
ValueError: Timezone offset does not match system offset: -7200 != -10800. Please, check your config files.
```

**To Reproduce**
1. Open cmd as administrator
2. Running: C:\Bazarr\WinPython\python-2.7.13\python.exe "C:\Bazarr\bazarr.py" --release-update --config "C:\ProgramData\Bazarr"

**Software**
 - Bazarr: today (master branch)
 - OS: Windows 10 build 1903

**Comment by morpheus65535** [comment]
Please confirm your Bazarr version as this bug has been fixed a long time ago: https://github.com/morpheus65535/bazarr/issues/405

**Comment by tsouza85** [comment]
v0.8.2.4

**Comment by rodrigoscoelho** [comment]
same here

**Comment by ltcarneiro** [comment]
Same here

**Comment by ltcarneiro** [comment]
It's broken. Wont start. Please, fix it. Thanks.

**Comment by morpheus65535** [comment]
I will, when I'm gonna get time to. Until then, sorry.

**Comment by tsouza85** [comment]
Brazilian Daylight Saving Time would start yesterday but has been canceled.  
I changed from UTC-3 to UTC-0 on Windows and it work, but I have incorrect time on Windows System...

**Comment by rodrigoscoelho** [comment]
I did the same. The problem has relation with the Brazilian daylight time,
that was canceled.

Enviado utilizando um Galaxy S9...

Em ter, 22 de out de 2019 01:19, Tiago Souza <notifications@github.com>
escreveu:

**Comment by morpheus65535** [comment]
Please insure your Windows 10 is up-to-date: https://techcommunity.microsoft.com/t5/Daylight-Saving-Time-Time-Zone/Time-zone-updates-for-Brazil-are-available/ba-p/751175

**Comment by rodrigoscoelho** [comment]
Updated here. Same problem

Enviado utilizando um Galaxy S9...

Em ter, 22 de out de 2019 13:37, morpheus65535 <notifications@github.com>
escreveu:

**Comment by morpheus65535** [comment]
@rodrigoscoelho Are you able to replace the `pytz` directory in `bazarr/libs` with the one in the latest release zip of pytz Python module? The `pytz`directory in the zip archive is located under `src` directory. You can get it from https://github.com/stub42/pytz/archive/release_2019.3.zip

Restart Bazarr and confirm if it fix your issue. Thanks!

**Comment by rodrigoscoelho** [comment]
Didn't work.

Enviado utilizando um Galaxy S9...

Em ter, 22 de out de 2019 14:27, morpheus65535 <notifications@github.com>
escreveu:

**Comment by rodrigoscoelho** [comment]
We are on gmt -3 on Brazil, which is equivalent to minus 10800 seconds.
When on summertime, it is only gmt -2, equivalent to minus 7200 seconds.
So, we are not on summertime, windows is correct, but the python is
expecting the summertime,causing the time difference. It should be only a
warning and this couldn't be a critical exception, IMHO.

Where can we force the timezone on configuration of bazarr? Is it possible?

Thanks

Enviado utilizando um Galaxy S9...

Em ter, 22 de out de 2019 20:29, rodrigoscoelho <notifications@github.com>
escreveu:

**Comment by jethromorais** [comment]
Same error here, I've downloaded the pytz from https://pypi.org/project/pytz/ ( latest version )

pip install pytz
Collecting pytz
 Downloading https://files.pythonhosted.org/packages/e7/f9/f0b53f88060247251bf481fa6ea62cd0d25bf1b11a87888e53ce5b7c8ad2/pytz-2019.3-py2.py3-none-any.whl (509kB)
    100% |UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU| 512kB 3.2MB/s
Installing collected packages: pytz
Successfully installed pytz-2019.3

I've copied the files from C:\Python27\Lib\site-packages\pytz\zoneinfo\Brazil to C:\bazarr\libs\pytz\zoneinfo\Brazil , I do believe the problem lies with the cancellation of daylight savings in Brazil...
the problem persist.

**Comment by dkage** [comment]
I think this is a problem for every single brazilian user. Daylight saving time was cancelled by the president some months ago, but systems are still configured waiting for it to happen.

Got the same exact error.

**Comment by dkage** [comment]
From my understanding about the problem, **pytz** needs to be updated with the new information about the non usage of daylight saving time in Brazil.

As a workaround to remove the problem on my system to keep using Bazarr for the time being, I changed the definition of function **get_tz_offset**, located in file:

> Bazarr\bazarr\utils.py

Changed function from:

```
def get_tz_offset(tz):
    """Get timezone's offset using built-in function datetime.utcoffset()."""
    return int(datetime.datetime.now(tz).utcoffset().total_seconds())
```

to:

```
def get_tz_offset(tz):
    """Get timezone's offset using built-in function datetime.utcoffset()."""
    if str(tz) == 'America/Sao_Paulo':
        return int(-10800)
    else:
        return int(datetime.datetime.now(tz).utcoffset().total_seconds())
```

This way I "force" Bazarr to set the offset value to -10800 instead of receiving that value from pytz.

Don't know if it's for the better to remove the **raise** error there on the if statement that checks if values of offsets are equal  (_and leave the print error as just a warning_), or just wait for a new pytz version to be used by bazarr (_don`t actually know if it's already using the lastest version_).

**Comment by morpheus65535** [comment]
Thanks to @dkage for fixing this issue!

**Comment by rodrigoscoelho** [comment]
I don't think he fixed the issue. He forced a wrong value to fix the issue now. When the PYTZ corrects the data for Brazil, the issue probably will rise back.  Not?

**Comment by morpheus65535** [comment]
His last comment here was a temporary fix that didn't get pushed to Bazarr. He fixed it in PR #646 where he updated pytz and tzlocal to latest version.

**Comment by lgabs** [comment]
I was passing through the same problem in a different context, so I did changes suggested by @dkage and it worked perfectly. Thanks!
